jcl/src/openj9.traceformat/share/classes/com/ibm/jvm/traceformat/TraceFormat.java

```diff
@@ -355,21 +356,23 @@ public static void main(String[] args) throws Exception {
 				thread = current;
 				output.println(formatted.toString());
 			}
-			
+
 			/* print percentage */
 			if (context.getTotalRecords() != recordsProcessed) {
 				recordsProcessed = context.getTotalRecords();
 				float processedMbytes = ((float) (context.getTotalRecords() * context.getRecordSize())) / (1024 * 1024);
```

Comment:
This can be improved; we can reuse `recordsProcessed` and capture the number of bytes processed for reuse below:
```
	long bytesProcessed = recordsProcessed * context.getRecordSize();
	float processedMbytes = bytesProcessed / (1024.0f * 1024.0f);
```
Note also that by using float constants, the explicit cast is not needed.

Reply:
updated.